shuffle/README.md

```diff
@@ -0,0 +1,32 @@
+# Fisher-Yates Shuffle function test vectors
+
+We offer test vectors for the implementation of the shuffle function as described in Appendix F of the Gray Paper.
+This section presents two definitions, a recursive one that uses an integer sequence as random seeds in Eq 329, and one (Eq 331) which takes a 32-byte hash as entropy, and is defined in terms of the previous one.
```

Comment:
If you're going to refer to equation numbers, you should also include a version as these numbers may change from version to version.